tests/integration/tc_71_18.c

```diff
@@ -22,24 +22,33 @@ char dev2_header[] = "Device No. 1:\ttype = SimpleTestDevice, uid = 17185, itype
 char custom_dev_header[] = "Device No. 2:\ttype = CustomData, uid = 2020, itype = 32\n";
 
 // parameters names we look for
-char increasing[] = "\"INCREASING\"";
-char doubling[] = "\"DOUBLING\"";
-char flipflop[] = "\"FLIP_FLOP\"";
-char myint[] = "\"MY_INT\"";
+char increasing[] = "INCREASING";
+char doubling[] = "DOUBLING";
+char flipflop[] = "FLIP_FLOP";
+char myint[] = "MY_INT";
+
+#define NUM_PARAMS 4
+
+char* parameters[NUM_PARAMS] = {
+    increasing,
+    doubling,
+    flipflop,
+    myint,
+};
 
 /**
  * Helper function to send dev1_params subscriptions
  * to device 1 and dev2_params subscriptions to device 2
```

Comment:
Add a comment mentioning that it returns a DeviceData message that *should* reflect the new subscription, and that this message should be checked

Reply:
Added